backend/app/views/spree/admin/reports/_filters.html.erb

```diff
@@ -8,16 +8,22 @@
             <i class="icon icon-chevron-left"></i>
           </label>
         </div>
-        <select class="custom-select js-filter-node" id="reports-time-period" data-param="period">
-          <option value="today">Today</option>
-          <option value="yesterday">Yesterday</option>
-          <option selected value="this_week">This Week</option>
-          <option value="last_week">Last Week</option>
-          <option value="this_month">This Month</option>
-          <option value="last_month">Last Month</option>
-          <option value="this_year">This Year</option>
-          <option value="last_year">Last Year</option>
-        </select>
+        <%= select_tag "reports-time-period",
+          options_for_select(
+            [[ "Today", "today" ],
```

Comment:
@devilcoders let's use translations for this

Reply:
@damianlegawiec This will be changed in final version of PR